OneBlink.SDK/models/Form.cs

```diff
@@ -20,7 +20,12 @@ public Form()
             long? id = null,
             string postSubmissionAction = "FORMS_LIBRARY",
             bool isAuthenticated = true,
+            List<FormSubmissionEvent> draftEvents = default(List<FormSubmissionEvent>),
             List<FormSubmissionEvent> submissionEvents = default(List<FormSubmissionEvent>),
+            List<FormSubmissionEvent> paymentEvents = default(List<FormSubmissionEvent>),
+            List<FormSubmissionEvent> schedulingEvents = default(List<FormSubmissionEvent>),
+            List<FormApprovalFlowInstanceStep> approvalSteps = default(List<FormApprovalFlowInstanceStep>),

```

Comment:
I dont think class has the correct properties. Maybe just confirm with the types repo.

Reply:
This should be a class that only has `group`, `label`, `isConditional, `requiresAllConditionalPredicates`, and `conditionalPredicates`.

Reply:
I recommend `FormApprovalStep` as the class name.